applications/crossbar/priv/couchdb/schemas/system_config.kazoo_endpoint.json

```diff
@@ -25,6 +25,11 @@
             "description": "kazoo_endpoint default ignore completed elsewhere",
             "type": "boolean"
         },
+        "default_rtcp_mux": {
```

Comment:
@sergey-safarov because this is new functionality, this should be opt-in, and thus should default to 'false', no?